### PR TITLE
Add missing space to `profile-gists-link`

### LINKED ISSUE #4751: Gist profile tab count lacks margin
There's little space on the left side of the Gist count

Screen Shot 5

The same issue might apply to other tabs we add, we should also check them (if we have any)

### Example URL

https://github.com/xt0rted

### Browser(s) used

Safari

### PR DESCRIPTION
**Pull request by kidonng**
Fix #4751 

## Test URLs

https://github.com/xt0rted

## Screenshot

Before: 

image

After:

image

### CODE REVIEW

**Review by fregante — APPROVED** [review]

File: source/features/profile-gists-link.tsx [hunk 1]
```diff
@@ -44,6 +45,5 @@ void features.add(__filebasename, {
 	include: [
 		pageDetect.isUserProfile,
 	],
-	awaitDomReady: false,
```
Comment (yakov116) [inline source/features/profile-gists-link.tsx:47]: Why was this removed?

Comment (kidonng) [reply source/features/profile-gists-link.tsx:47]: https://github.com/sindresorhus/refined-github/issues/4751#issuecomment-912606783

Comment (yakov116) [reply source/features/profile-gists-link.tsx:47]: So then the elementReady should be removed
